x-pack/plugins/security_solution/common/endpoint/data_generators/endpoint_action_generator.ts

```diff
@@ -88,7 +88,7 @@ export class EndpointActionGenerator extends BaseDataGenerator {
         output = {
           type: 'json',
           content: {
-            code: 'ra_get-file_success',
+            code: 'ra_get-file_success_done',
```

Comment:
What's the difference between `succecss` and `success_done`?

Reply:
this is also an interesting magic string. what are the other possible values? Do we check and do anything with these on the application end?

Reply:
I made up the initial `code` based on the pattern the endpoint uses. I just updated it based on the live testing Kevin yesterday after seeing what the endpoint was populating it with.

There might be other codes, especially for failures. I have a note to reach out to Endpoint team once they are a little further along with their dev to get the list of them. The codes will be helpful in failure conditions to provide better UX messaging in the error message displayed to users (we do something like this now for `kill`/`suspend-process`.

Reply:
:+1: 

when we do get the codes, we'll have a group of consts or other way to enumerate the possibilities, yeah?

Reply:
@pzl soon. I have to reach out on it.
We'll probably keep them here: 

https://github.com/elastic/kibana/blob/cb4d6aa8d50d02aa52bfbd32d5cac265522d5d0e/x-pack/plugins/security_solution/public/management/components/endpoint_responder/endpoint_action_response_codes.ts